### PR TITLE
Сonnection cli arguments

### PR DESCRIPTION
**Pull request by reponche**

### CODE REVIEW

File: weatherbot/get_temp.py [hunk 1]
```diff
@@ -14,4 +12,4 @@ def conv_kelv_to_cels(temp):
     return round(cels)
 
 def add_C(cels):
-    return (cels + "°C").encode('utf-8')
+    return (str(cels) + "°C").encode('utf-8')
```
Comment (chuwy) [inline weatherbot/get_temp.py]: Just in case, what you really want here is `return str(cels) + u"\xb0C"`

File: weatherbot/main.py [hunk 2]
```diff
@@ -9,49 +9,19 @@
...
-
+    if len(sys.argv) == 2:
+        program_name = sys.argv[0]
+        city = sys.argv[1]
+        weather = get_weather(city, OWN_TOKEN)
+        temp = conv_kelv_to_cels(weather)
+        cels = add_C(temp)
+        print(cels)
+    else:
+        print("Please, enter the city")
```
Comment (chuwy) [inline weatherbot/main.py]: What happens if I provide more? E.g. ` python3 -m weatherbot.main Krasnoyarsk Tomsk`? It tells me `Please, enter the city`, but I provided the city - I think error message can be more specific.

File: weatherbot/main.py [hunk 3]
```diff
@@ -9,49 +9,19 @@
...
-            send_message(chat_id, "Hi, I can help you to know the weather, just tell me the city.")
-        elif '/city' in text:
-            send_message(chat_id, get_weather())
-        else:
-            send_message(chat_id, "If you want to know the weather just send me a city.")
-
+    if len(sys.argv) == 2:
+        program_name = sys.argv[0]
+        city = sys.argv[1]
+        weather = get_weather(city, OWN_TOKEN)
```
Comment (chuwy) [inline weatherbot/main.py]: You don't need this temporary `city` variable. You can pass `sys.argv[1]` straight to the `get_weather` funciton.

File: weatherbot/main.py [hunk 4]
```diff
@@ -9,49 +9,19 @@
...
-
-        if '/start' in text:
-            send_message(chat_id, "Hi, I can help you to know the weather, just tell me the city.")
-        elif '/city' in text:
-            send_message(chat_id, get_weather())
-        else:
-            send_message(chat_id, "If you want to know the weather just send me a city.")
-
+    if len(sys.argv) == 2:
+        program_name = sys.argv[0]
```
Comment (chuwy) [inline weatherbot/main.py]: This variable is not used anywhere.

File: weatherbot/main.py (hunk 2 above)
Comment (reponche) [reply weatherbot/main.py]: Thx! I will fix it.

File: weatherbot/main.py (hunk 3 above)
Comment (reponche) [reply weatherbot/main.py]: Thx! I will fix it.

File: weatherbot/main.py (hunk 4 above)
Comment (reponche) [reply weatherbot/main.py]: Thx! I will fix it.

File: weatherbot/get_temp.py (hunk 1 above)
Comment (reponche) [reply weatherbot/get_temp.py]: Thank you, I fixed it.

File: weatherbot/main.py [hunk 5]
```diff
@@ -13,15 +11,25 @@
...
-    if len(sys.argv) == 2:
-        program_name = sys.argv[0]
-        city = sys.argv[1]
-        weather = get_weather(city, OWN_TOKEN)
-        temp = conv_kelv_to_cels(weather)
-        cels = add_C(temp)
-        print(cels)
-    else:
+    try:
+        input = int(sys.argv[1])
```
Comment (chuwy) [inline weatherbot/main.py]: Why exactly do we need this?

**Pull request merged**